src/api/config.rs

```diff
@@ -516,6 +525,10 @@ impl SpeedSettings {
   const fn enable_inter_tx_split_preset(speed: usize) -> bool {
     speed >= 9
   }
+
+  const fn fine_directional_intra_preset(speed: usize) -> bool {
+    speed <= 1
```

Comment:
```suggestion
    speed <= 1 || speed >= 6
```

Based on our tests, the performance penalty is not bad for speed 6 and up, so we can leave it enabled for higher speed levels too.

Reply:
Fixed it and update the title as above.

Reply:
Great, thanks!